api_tests/applications/views/test_application_detail.py

```diff
@@ -1,35 +1,41 @@
+import pytest
 import mock
 
-from nose.tools import *  # flake8: noqa
-
 from website.models import ApiOAuth2Application, User
 from website.util import api_v2_url
-
 from tests.base import ApiTestCase
+from api_tests.utils import assert_dict_contains_subset
 from osf_tests.factories import ApiOAuth2ApplicationFactory, AuthUserFactory
 
 
 def _get_application_detail_route(app):
-    path = "applications/{}/".format(app.client_id)
+    path = 'applications/{}/'.format(app.client_id)
     return api_v2_url(path, base_route='/')
 
 
 def _get_application_list_url():
-    path = "applications/"
+    path = 'applications/'
     return api_v2_url(path, base_route='/')
 
+@pytest.fixture()
+def user():
+    return AuthUserFactory()
 
-class TestApplicationDetail(ApiTestCase):
-    def setUp(self):
-        super(TestApplicationDetail, self).setUp()
+@pytest.mark.django_db
+@pytest.mark.usefixtures('app')
```

Comment:
I think this mark is unnecessary, since you're declaring `app` in the method signatures.